robot_upstart/install_script.py

```diff
@@ -50,16 +50,22 @@ def get_argument_parser():
     p.add_argument("--job", type=str,
                    help="Specify job name. If unspecified, will be constructed from package name (first " +
                    "element before underscore is taken, e.g. 'myrobot' if the package name is 'myrobot_bringup').")
+    p.add_argument("--rmw", type=str, metavar="rmw_fastrtps_cpp",
+                   help="Specify RMW DDS being used. rmw_fastrtps_cpp or rmw_cyclonedds_cpp")
+    p.add_argument("--fastrtps_config", type=str,
```

Comment:
Could `--fastrtps_config` and `--cyclonedds_config` be merged into one variable?